src/TerminalFighter/TargetingSystem/TargetingSystem.cpp

```diff
@@ -2,13 +2,20 @@
 #include <cstdlib>
 #include <ctime>
 
+#include <SDL_ttf.h>
+
 #include "TargetingSystem.h"
+#include "GraphicsHandler/I_GraphicsHandler.h"
 
 TargetingSystem::TargetingSystem(int word_length_lower_bound, int word_length_upper_bound, std::string color_hex)
     : word_length_lower_bound_(word_length_lower_bound)
     , word_length_upper_bound_(word_length_upper_bound)
+    , hitbox_(Hitbox(0, 0, 1920, 1080)) //hardcoded numbers, TODO, don't have these hardcoded
     , color_hex_(color_hex) {
     setup_local_dict("assets/dictionary.txt");
+    srand(time(NULL)); //TODO this should be called at a higher level, maybe Universe //this actually makes rand() be random
+    default_font_ = TTF_OpenFont("/usr/share/fonts/truetype/ubuntu-font-family/Ubuntu-B.ttf", 24);
+    white_ = {255, 255, 255};
```

Comment:
I don't feel like this needs to be a member variable, more like a constant. This shouldn't change?